packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -255,6 +258,17 @@ class CheckboxListTile extends StatelessWidget {
   /// If non-null, defines the background color when [CheckboxListTile.selected] is true.
   final Color? selectedTileColor;
 
+  /// Defines how compact the list tile's layout will be.
+  ///
+  /// {@macro flutter.material.themedata.visualDensity}
+  final VisualDensity? visualDensity;
+
+  /// {@macro flutter.widgets.Focus.focusNode}
+  final FocusNode? focusNode;
+
+  /// If specified, defines the feedback property for `CheckboxListTile`.
```

Comment:
This comment doesn't explain what setting it to true, false, or null would do. Can you elaborate?

Reply:
And the same for the other two...

Reply:
I have added a macro template to define the `enableFeedback` property.